File: IQ/Classes/clsLimit.vb
Public Class clsLimit

    '[QtyInstalled],[QtyMax],[Incr_Min],[Incr_Pref]
    Property Qinstalled As Integer
    Property Qmin As Integer
    Property Qmax As Integer
    Property MinIncr As Integer
    Property PrefIncr As Integer

    Public Sub New(Installed As Integer, Qmin As Integer, Qmax As Integer, MinIncr As Integer, PrefIncr As Integer)

        Me.Qinstalled = Installed
        Me.Qmin = Qmin
        Me.Qmax = Qmax
        Me.MinIncr = MinIncr
        Me.PrefIncr = PrefIncr

    End Sub

End Class

File: IQ/Classes/clsLimit.cs
public class clsLimit
{

    //[QtyInstalled],[QtyMax],[Incr_Min],[Incr_Pref]
    public int Qinstalled { get; set; }
    public int Qmin { get; set; }
    public int Qmax { get; set; }
    public int MinIncr { get; set; }
    public int PrefIncr { get; set; }

    public clsLimit(int Installed, int Qmin, int Qmax, int MinIncr, int PrefIncr)
    {

        this.Qinstalled = Installed;
        this.Qmin = Qmin;
        this.Qmax = Qmax;
        this.MinIncr = MinIncr;
        this.PrefIncr = PrefIncr;

    }

}